pkgs/hooks_runner/lib/src/model/kernel_assets.dart

```diff
@@ -40,7 +41,7 @@ class KernelAssets {
       },
     };
 
-    return yamlEncode(yamlContents);
+    return const JsonEncoder.withIndent('  ').convert(yamlContents);
```

Comment:
I can see a future reader stumbling over this line and wondering why we encode yaml with the Json encoder. Can you leave a comment or fix the naming?